configs/tc.prod.yml

```diff
@@ -1,9 +1,154 @@
 # Example of a production config for Taskcluster
+#
+# See the training guide for more information:
+#   https://mozilla.github.io/firefox-translations-training/training-guide.html
+#
+# The defaults for these parameters are available at:
+#   taskcluster/translations_taskgraph/parameters.py
+
+# An "experiment" is an individual training run.
+experiment:
+  # Provide an identifiable name for your experiment.
+  name: baseline_en_ru
+
+  # The source and target languages. This is the language tag part of the
+  # BCP 47 locale identifier.
+  src: en
+  trg: ru
+
+  # The metric to use for computing the best model validation.
+  #   cross-entropy, ce-mean-words, perplexity, valid-script, translation, bleu,
+  #   bleu-segmented, chrf
+  # See: https://github.com/marian-nmt/marian/blob/65bf82ffce52f4854295d8b98482534f176d494e/src/common/config_parser.cpp#L588-L592
+  best-model: chrf
+
+  # Use the Opus Cleaner tool on the data cleaning step.
+  # https://github.com/hplt-project/OpusCleaner
+  use-opuscleaner: "true"
+
+  # Bicleaner is a tool that aims at detecting noisy sentence pairs in a parallel corpus.
+  # It indicates the likelihood of a pair of sentences being mutual translations (with a
+  # value near to 1) or not (with a value near to 0). Sentence pairs considered very
+  # noisy are scored with 0. Bicleaner AI will be used first if the language is
+  # available, otherwise it will fallback to the original non-AI Bicleaner.
+  #
+  # See:
+  #   https://github.com/bitextor/bicleaner-ai
+  #   https://github.com/bitextor/bicleaner
+  #
+  # For supported languages see:
+  #   https://github.com/bitextor/bicleaner-ai-data/releases
+  #   https://github.com/bitextor/bicleaner-data/releases
+  #
+  # New language releases are added to: taskcluster/ci/fetch/bicleaner.yml
+  bicleaner:
+    default-threshold: 0.5
+    dataset-thresholds:
+      opus_CCAligned/v1: 0.7
+      opus_LinguaTools-WikiTitles/v2014: 0.7
+      opus_OpenSubtitles/v2018: 0.8             # Example of a higher filtering level.
+      opus_ParaCrawl/v9: 0.0                    # Example of cleaning being disabled.
+      opus_WikiMatrix/v1: 0.7
+      opus_bible-uedin/v1: 0.7
+
+  # Limits the maximum sentences use in the monolingual data for both the source and target
+  # languages.
+  mono-max-sentences-src: 100_000_000
+  mono-max-sentences-trg: 20_000_000
+
+  # The number of sentences split out into separate files for the monolingual data.
+  # Also see taskcluster.split-chunks below.
+  split-length: 2_000_000
+
+  # How many bytes of the corpus are sampled to be used in SentencePiece
+  # tokenization training.
+  spm-sample-size: 10_000_000
+
+  # How big the vocab will be. The vocab is shared between the source and the
```

Comment:
It's unlikely we'll need to train something with less than 5M sentences, so the vocab size will likely never change. I recommend moving this extra info to the training guide. It will allow adding proper links and reduce size of this config.

Reply:
Ok, seems reasonable. I'll document the reasoning too. This was the source of one of my failures in training en-ca, since the pipeline script failed for me. 32,000 seems like a good default to me.